src/cobalt.rs

```diff
@@ -139,13 +152,18 @@ fn generate_doc(posts_data: &[liquid::Value], doc: &mut Document, context: &Cont
         "page".to_owned(),
         liquid::Value::Object(doc.attributes.clone()),
     );
-    let doc_html = doc.render(&globals, &context.liquid, &context.layouts)
+    let doc_html = doc
+        .render(&globals, &context.liquid, &context.layouts)
         .chain_err(|| format!("Failed to render for {:?}", doc.file_path))?;
+    trace!(
+        "doc.file_path: {:?}",
```

Comment:
Is this a temp debug aid you plan to remove?

Reply:
oops, yes it's a forgotten debug log